src/lib/globals.ts

```diff
@@ -5,6 +5,12 @@ declare global {
       postMessage: (msg: string) => void
     }
   }
+  interface String {
+    split<Separator extends string | RegExp, Limit extends number>(
+      separator: Separator,
+      limit?: Limit,
+    ): (string | undefined)[]
```

Comment:
The actual safety

Reply:
See https://github.com/shapeshift/web/pull/6811/files#r1591577558 , we can use typescript for this.

We likely have many occurrences where we're accessing arrays with missing values, not just in the case of `String.split()`, but the sheer number of them might mean we're best off starting here and following up with the blanket rule.

Reply:
Ah, I see your original comment

> since we can't do the `noUncheckedIndexedAccess` dance without having 500+ errors, this at least brings strictness for the specific

Reply:
🧠